WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -216,12 +251,106 @@ public void onResponse(JSONObject response) {
+                                        AnalyticsTracker.Stat.SITE_SETTINGS_SAVED_REMOTELY, mSite, properties);
+                            }
+                        }
+                    }, new RestRequest.ErrorListener() {
+                        @Override
+                        public void onErrorResponse(VolleyError error) {
+                            AppLog.w(AppLog.T.API, "Error POSTing site settings changes: " + error);
+                            notifySavedOnUiThread(error);
+                        }
+                    });
+        } catch (JSONException exception) {
+            AppLog.w(AppLog.T.API, "Error serializing settings changes: " + exception);
+            notifySavedOnUiThread(exception);
+        }
+    }
+
+    private void pushJetpackProtectAndSsoSettings() {
+        final Map<String, Object> params = serializeJetpackProtectAndSsoParams();
+        if (params.isEmpty()) {
+            return;
+        }
+        WordPress.getRestClientUtilsV1_1().setJetpackSettings(mSite.getSiteId(), params,
+                new RestRequest.Listener() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        AppLog.d(AppLog.T.API, "Jetpack settings updated");
+                        mRemoteJpSettings.monitorActive = mJpSettings.monitorActive;
+                        mRemoteJpSettings.jetpackProtectEnabled = mJpSettings.jetpackProtectEnabled;
+                        mRemoteJpSettings.jetpackProtectWhitelist.clear();
+                        mRemoteJpSettings.jetpackProtectWhitelist.addAll(mJpSettings.jetpackProtectWhitelist);
+                        mRemoteJpSettings.ssoActive = mJpSettings.ssoActive;
+                        mRemoteJpSettings.ssoMatchEmail = mJpSettings.ssoMatchEmail;
+                        mRemoteJpSettings.ssoRequireTwoFactor = mJpSettings.ssoRequireTwoFactor;
+                        notifySavedOnUiThread(null);
+                    }
+                }, new RestRequest.ErrorListener() {
+                    @Override
+                    public void onErrorResponse(VolleyError error) {
+                        AppLog.w(AppLog.T.API, "Error updating Jetpack settings: " + error);
+                        notifySavedOnUiThread(error);
+                    }
+                });
+    }
+
+    private void pushJetpackMonitorSettings() {
+        final Map<String, String> params = serializeJetpackMonitorParams();
+        if (params.isEmpty()) {
+            return;
+        }
+        WordPress.getRestClientUtilsV1_1().setJetpackMonitorSettings(
+                mSite.getSiteId(), params, new RestRequest.Listener() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        AppLog.d(AppLog.T.API, "Jetpack Monitor module updated");
+                        mRemoteJpSettings.emailNotifications = mJpSettings.emailNotifications;
```

Comment:
Same as [this](https://github.com/wordpress-mobile/WordPress-Android/pull/6440#discussion_r134191075)